02-create.md

```diff
@@ -17,7 +17,7 @@ $ mkdir planets
 $ cd planets
 ~~~
 
-and tell Git to make it a [repository](reference.html#repository)&mdash;a place where
+and tell Git to make it a [repository](reference.html#repository) &ndash; a place where
```

Comment:
But it's up to you: either revert this back to the em-dash or use a comma, please.